src/Uno.UI/UI/Xaml/DependencyProperty.Dictionary.FrameworkPropertiesForType.cs

```diff
@@ -13,18 +13,19 @@
 using Uno.Collections;
 
 using _Key = Uno.CachedTuple<System.Type, Windows.UI.Xaml.FrameworkPropertyMetadataOptions>;
+using System.Collections;
 
 namespace Windows.UI.Xaml
 {
 	public sealed partial class DependencyProperty
 	{
 		private class FrameworkPropertiesForTypeDictionary
```

Comment:
This dictionary has a single static instance that is kept for the lifetime of the whole app.